torba/baseaccount.py

```diff
@@ -175,3 +175,38 @@ def get_private_key(self, chain, index):
 
     def get_balance(self):
         return self.ledger.db.get_balance_for_account(self)
+
+
+class SingleKeyAccount(BaseAccount):
+    def __init__(self, ledger, seed, encrypted, private_key,
+                 public_key, receiving_gap=0, change_gap=0,
+                 receiving_maximum_use_per_address=0,
+                 change_maximum_use_per_address=0):
+        self.ledger = ledger
+        self.seed = seed
+        self.encrypted = encrypted
+        self.private_key = private_key
+        self.public_key = public_key
+
+        self.receiving, self.change = self.keychains = (
+            # (account, parent_key, chain_number, gap, maximum_use_per_address)
+            KeyChain(self, public_key, 0, 0, 0),
```

Comment:
What are sane defaults for `gap`, `maximum_use_per_address`?

Reply:
I would make a new KeyChain like `SingleKeyChain` that always returns the same key.